1-exercises/B-highlight-words/script.js

```diff
@@ -1,5 +1,28 @@
 function highlightWords(paragraph, colours) {
   // Write your code here...
+  let divElement = document.querySelector("#content");
+
+  let getParagraphs = document.createElement("p");
+  divElement.appendChild(getParagraphs)
+
+  let selectElement = document.createElement("select");
+  getParagraphs.appendChild(select)
+
+  colours.forEach((colour) => {
+    let optionElement = document.createElement("option");
+    optionElement.value = colour;
+    optionElement.innerHTML = colour;
+    selectElement.appendChild("optionElement");
+  });
+
+let pElement = paragraph.split("");
+
+words.forEach((x) => {
+  let span= document.createElement("span");
+  span.innerText = x + " ";
+  pElement.appendChild(span);
+});
```

Comment:
Line 11 and 20 - great use of forEach and arrow functions.  This is nice coding.
There are a number of bugs though so don't forget use the inspect tool in your browser to find problems - for example

```
script.js:9 Uncaught ReferenceError: select is not defined
    at highlightWords (script.js:9:29)
    at script.js:35:1
```
Look at line 9 again - appendChild(select) should be appendChild(selectElement).  But you also need to append to the content (`divElement`) not the `getParagraphs` .   

Take care in using all the information you are given too. So it says...

_You'll need to turn the paragraph into an array to iterate over it. You can use the ``.split(" ")`_

Your code has paragraph.split("")  so you wont split into an array of words. You need `" "` not `""`